src/components/Home/UpcomingEvents.tsx

```diff
@@ -331,7 +331,7 @@ const UpcomingEvents = ({ className }:Props) => {
 
 			return (
 				<Tooltip color='#E5007A' title={eventList}>
-					<Badge count={value.format('D')} color='transparent' className='bg-pink-400 rounded-full absolute -ml-3 mt-[-22px] w-full' />
+					<Badge count={value.format('D')} color='transparent' className='bg-pink-400 rounded-full absolute -ml-3 mt-[-22px]' />
```

Comment:
found a very simple solution, don't use Badge component instead of that use a div, 
```
<div className='...'>
    {value.format('D')}
</div>
```